internal/painter/gl/gl_core.go

```diff
@@ -40,6 +41,19 @@ func newTexture() Texture {
 	return Texture(texture)
 }
 
+func getTexture(object fyne.CanvasObject, creator func(canvasObject fyne.CanvasObject) Texture) (Texture, error) {
+	texture, ok := textures[object]
+
+	if !ok {
+		texture = creator(object)
+		textures[object] = texture
+	}
+	if texture == NoTexture {
```

Comment:
This does not seem like a change - the code pulled out from gl_common is essentially the same but you are creating an eror here instead of the common code?

Reply:
I am actually changing the common code to use an error. The issue is that the wasm backend can not compare texture like that as they are js.Value and == does not work on them /o\. Maybe I should split this change in two step. One modifying core to get an error and then use that change in the web backend.

Reply:
But your code gets to define NoTexture - can’t a suitable sentinel value be of type js.Value?

Reply:
It was possible before with 1.13, but with 1.14 and forward you just can't compare two js.Value without using Equal. The problem is that in the case of a structure with a js.Value inside it, you can not compare the structure itself. I think we could do manual Equal on each members, but then it break portability also. So returning an error did seems like the cleaner solution here.